etc/profile-a-l/Beyond-All-Reason.profile

```diff
@@ -0,0 +1,42 @@
+# Firejail profile for Beyond All Reason
+# Description: Open source real time strategy game
+# This file is overwritten after every install/update
+# Persistent local customizations
+include beyondallreason.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/Beyond All Reason
```

Comment:
This is where settings/gamerecords/mods/... are saved? Add it to `disable-prorams.inc`.

Reply:
It contains everything, including code/data from game updates and patches. So there needs to be the ability to execute shared objects etc